build.sbt

```
@@ -18,6 +23,7 @@ val `fs2-core`        = Def.setting("co.fs2"          %%% "fs2-core"        % V.
 val `fs2-io`          = Def.setting("co.fs2"          %% "fs2-io"           % V.fs2)
 val kittens           = Def.setting("org.typelevel"   %%% "kittens"         % V.kittens)
 val refined           = Def.setting("eu.timepit"      %%% "refined"         % V.refined)
+val refined211        = Def.setting("eu.timepit"      %%% "refined"         % V.refined211)
```

suggestion: remove line above and change this to
```suggestion
val refined = Def.setting {
  CrossVersion.partialVersion(scalaVersion.value) match {
    case Some((2, 11)) => "eu.timepit" %%% "refined" % V.refined211
    case _             => "eu.timepit" %%% "refined" % V.refined
  }
}
```
then you could revert all relevant changes around this below

---

nice. I'll do it